recipes/isofit/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "isofit" %}
+{% set version = "1.1.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/adam-erickson/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 4947c0067e9d2d62d510d0bb1e0a343088bc199410cce9a8d8e688d1c558f053
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python >=3.5
+    - setuptools
+    - numpy >=1.11
+    - scipy >=1.1.0
+    - numba >=0.38.0
+    - matplotlib >=2.2.2
+    - scikit-learn >=0.19.1
+    - spectral >=0.19
+    - pytest >=3.5.1
+    - pep8 >=1.7.1
+    - python-xxhash >=1.2.0
+
+test:
+  imports:
+    - {{ name }}
+
+about:
+  home: https://github.com/{{ name }}/{{ name }}
+  license: Apache 2.0
```

Minor thing for consistency with other packages, SPDX identifiers have a dash here:
```suggestion
  license: Apache-2.0
```